monitoring/environments/scaleway-parca-demo/main.jsonnet

```diff
@@ -32,6 +32,14 @@ local prometheuses = [
   }) {
     local p = self,
 
+    namespace: {
+      apiVersion: 'v1',
+      kind: 'Namespace',
+      metadata: {
+        name: 'parca-analytics',
```

Comment:
Namespaces are managed in [cluster-config](https://github.com/parca-dev/demo-deployments/tree/main/cluster-config)